src/licensedcode/data/rules/umich-merit_1.RULE

```diff
@@ -0,0 +1,25 @@
+---
+license_expression: umich-merit
+is_license_text: yes
+minimum_coverage: 95
+---
+
+[C] The Regents of the University of Michigan and Merit Network, Inc. 1992,
```

Comment:
See https://github.com/nexB/scancode-toolkit/issues/3659 for reference ./.. so we can keep this copyright for now.